docs/java/features/multi-tenancy/thread-context.md

```diff
@@ -0,0 +1,103 @@
+---
+id: multi-tenancy-thread-context
+title: Multi Tenancy with the Thread Context
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Thread Context
+custom_edit_url: https://github.com
+description: This article describes how the SAP Cloud SDK for Java provides an application context that is stored in a thread-safe manner and enables cloud native features to be used out of the box.
+keywords:
+- sap
+- cloud
+- sdk
+- thread context
+- multi tenancy
+- cloud native
+- tenant
+- user
+- principal
+- JWT
+- AuthToken
+image:
+---
+
+## What is a Thread Context?
+
+The SAP Cloud SDK for Java provides a so called `ThreadContext`. 
+It serves as thread-safe storage for potentially sensitive information. 
+Specifically the following three objects are stored:
+
+- The current _Tenant_
+- The current _Principal_ (User)
+- The [Jason Web Token](https://jwt.io) (JWT)
+
+This information is used throughout the SDK to provide features like tenant and principal isolation, JWT verification and authorization against other systems and services.
+To ensure different tenants and users are properly isolated in an application this information is always limited to the Thread it was created on, unless it is explicitly passed on by the application (see [Propagating the Thread Context]()).
```

Comment:
Will this be another brief material, I mean **Propagating the Thread Context**? Can we make a placeholder and sidebar link for it with just a header and TBD inside?

Reply:
Good catch, had forgotten to update it :D